.vscode/launch.json

```diff
@@ -37,7 +37,8 @@
         "--nolazy"
       ],
       "env": {
-        "NODE_ENV": "development"
+        "NODE_ENV": "development",
+        "TEST_FILTER": "^jsdoc_types$"
```

Comment:
Our test runner prints:

```
[...]
    ✓ import_from_goog (8490ms)
    - index_import
    - interface
    - jsdoc
    - jsdoc_types
    - jsdoc_types.untyped
    - jsx
    - methods
    - nullable
    - optional
    - parameter_properties
    - static
    - structural.untyped
    - super
    - type
    - type_and_value
    - typedef
    - typedef.untyped
    - underscore
    - variables
  1 passing (9s)
  44 pending
```

That seems clear enough? We can remove it again if somebody runs into it and gets confused?